src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/MessageBody.cs

```diff
@@ -234,12 +286,13 @@ protected virtual void OnConsumedBytes(int count)
             var connection = headers.HeaderConnection.ToString();
```

Comment:
Talked to @cesarbs and he points out that we don't split headers by commas so this will only matter when there are multiple `Connection` headers.